overlord/fdestate/fdestate.go

```diff
@@ -403,3 +404,106 @@ func MockSecbootGetPCRHandle(f func(devicePath, keySlot, keyFile string, hintExp
 		secbootGetPCRHandle = old
 	}
 }
+
+// KeyslotTarget uniquely identifies a target key slot by
+// its container role and name.
+type KeyslotTarget struct {
```

Comment:
maybe we should call it KeyslotRef , this is closer how we use Ref is other parts of snapd